.github/workflows/run_tests.yml

```diff
@@ -30,16 +30,17 @@ jobs:
     strategy:
       matrix:
         container:
-          # EOL April 2028 https://wiki.ubuntu.com/Releases
+          # EOL April 2023 https://wiki.ubuntu.com/Releases
```

Comment:
It's not clear to me whether we should stop supporting at "end of life" or "end of standard support", but I suppose the earlier "end of standard support" date is a good time to reconsider whether it's worth continuing to support.

Reply:
I think most people using Shadow won't be subscribed to Ubuntu's support program, and it would be a pain to test that version anyways since we'd have to somehow subscribe our CI to the program (I don't think we'd be eligible for a free subscription anyways). So I don't think we should support longer than the standard support date.